gguf-py/gguf/gguf.py

```diff
@@ -185,6 +185,19 @@ class MODEL_TENSOR(IntEnum):
         MODEL_TENSOR.FFN_DOWN:      "blk.{bid}.ffn_down",
         MODEL_TENSOR.FFN_UP:        "blk.{bid}.ffn_up",
     },
+    MODEL_ARCH.MPT: {
+        MODEL_TENSOR.TOKEN_EMBD:    "wte",
+        MODEL_TENSOR.OUTPUT_NORM:   "norm_f",
+        # note: MPT output is tied to (same as) wte in original model;
+        # for easier implementation in llama.cpp it's duplicated in GGUF, though :/
+        MODEL_TENSOR.OUTPUT:        "output",
+        MODEL_TENSOR.ATTN_NORM:     "blk.{bid}.norm_1",
+        MODEL_TENSOR.FFN_NORM:      "blk.{bid}.norm_2",
+        MODEL_TENSOR.ATTN_QKV:      "blk.{bid}.attn.Wqkv",
```

Comment:
The GGUF spec this should be attn_qkv, which would match GPT-NeoX. Similarly, the llama.cpp implementation should use the standard name.

Reply:
Wow, I really overlooked all the tensor name mappings. Fixed.

Reply:
By the way, I know this is confusing and I am changing this to make more sense in #3409.